Core/Core/Transports/ServerUtils/ServerAPI.cs

```diff
@@ -335,13 +325,9 @@ CbObjectDownloaded onObjectCallback
     childrenHttpMessage.Content = new StringContent(serializedPayload, Encoding.UTF8, "application/json");
     childrenHttpMessage.Headers.Add("Accept", "text/plain");
 
-    HttpResponseMessage childrenHttpResponse;
-    do
-    {
-      childrenHttpResponse = await _client
-        .SendAsync(childrenHttpMessage, HttpCompletionOption.ResponseHeadersRead, CancellationToken)
-        .ConfigureAwait(false);
-    } while (ShouldRetry(childrenHttpResponse));
+    HttpResponseMessage childrenHttpResponse = await _client
```

Comment:
1. Same question about retrying a a higher level here... and at the other places